quickblox.min.js

```diff
@@ -43786,6 +43822,10 @@ WebRTCSession.prototype.processCall = function(peerConnection, ext) {
     extension.opponentsIDs = this.opponentsIDs;
     extension.sdp = peerConnection.localDescription.sdp;
 
+    //TODO: set bandwidth to the userInfo object
```

Comment:
Do you need this TODO as comment?

Reply:
Later the bandwidth will have its own property in the extensions, but now we need to set it to userInfo, because iOS and the Android SDK do not have the bandwidth property.